source/include/gridrec/filters.h

```diff
@@ -41,70 +41,139 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#pragma once
+// Possible speedups:
+//   * Profile code and check adding SIMD to various functions (from OpenMP)
 
-#include <complex.h>
-#include <stdlib.h>
+//#define WRITE_FILES
+#define _USE_MATH_DEFINES
```

Comment:
```suggestion
#ifndef M_PI
#    define M_PI 3.14159265359
#endif
```
I think we are defining M_PI directly because not every compiler has `_USE_MATH_DEFINES`.